src/ServiceInsight/MessageList/MessageListView.xaml

```diff
@@ -12,20 +12,6 @@
              d:DesignHeight="200"
              d:DesignWidth="500"
              mc:Ignorable="d">
-    <UserControl.Resources>
-        <Style x:Key="FocusedRowStyle" TargetType="dxg:RowControl">
-            <Style.Triggers>
-                <DataTrigger Binding="{Binding Path=IsSelected}" Value="True">
-                    <Setter Property="Background" Value="{StaticResource FocusedRowBackgroundBrush}" />
-                    <Setter Property="Foreground" Value="{StaticResource FocusedRowForegroundBrush}" />
-                </DataTrigger>
-                <Trigger Property="dxg:DataViewBase.IsFocused" Value="True">
```

Comment:
Only this section needs to be removed. If you remove the whole style, the selection on the grid has the default blue coloring. The brushes, Rowstyle and IsSelected binding should stay

Reply:
Good catch. The resulting style should be like the following:

```
<Style x:Key="FocusedRowStyle" TargetType="dxg:RowControl">
  <Style.Triggers>
    <DataTrigger Binding="{Binding Path=IsSelected}" Value="True">
      <Setter Property="Background" Value="{StaticResource FocusedRowBackgroundBrush}" />
      <Setter Property="Foreground" Value="{StaticResource FocusedRowForegroundBrush}" />
    </DataTrigger>
  </Style.Triggers>
</Style>
```

I'd also suggest renaming the style to `SelectedRowStyle` and updating line 220 accordingly.

Reply:
I just restored it to the previous name as otherwise its a name refactoring.